### PR TITLE
Resolve all compiler warnings.

### PR DESCRIPTION
**Pull request by xxkat**

### CODE REVIEW

File: src/performance/Score.cpp [hunk 1]
```diff
@@ -47,6 +47,8 @@ void Score::AppendToUpdateBatch(UpdateBatch& batch) const
 		TotalValue(),
 		_scoreId
 	));
+
+	batch.AppendAndCommitNonThreadsafe(StrFormat("INSERT INTO score_process_queue (mode, score_id, status) VALUES ({0}, {1}, 1) ON DUPLICATE KEY UPDATE status = 1", static_cast<int>(_mode), _scoreId));
```
Comment (peppy) [inline src/performance/Score.cpp:51]: What is this?

**Review by peppy — COMMENTED** [review]
ignoring the weird issue, not sure that is a worthwhile effort as this project is working fine and not intended to last too much longer into the future.

File: src/performance/Score.cpp (hunk 1 above)
Comment (xxkat) [reply src/performance/Score.cpp:51]: image

I didn't notice but when I initially pulled, this came along with it.

Comment (stanriders) [reply src/performance/Score.cpp]: I believe you pulled from a wrong repo, your fork seem to be based from peppy/osu-perf instead of ppy/osu-perf

Comment (xxkat) [reply src/performance/Score.cpp]: oh my, my apologies, i mustn’t have been thinking straight.

**Pull request closed without merging**